### PR TITLE
Add docstring for DatasetFieldSchema.type

### PR DESCRIPTION
**Pull request by lbam**

### CODE REVIEW

File: src/schematools/types.py [hunk 1]
```diff
@@ -1093,6 +1093,12 @@ def required(self) -> bool:
 
     @property
     def type(self) -> str:
+        """Returns the type of this field.
```
Comment (jjmurre) [inline src/schematools/types.py:1096]: Maybe you can also point to the documentation at: https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#data-types

**Review by jjmurre — APPROVED** [review]
LGTM

File: src/schematools/types.py (hunk 1 above)
Comment (lbam) [reply src/schematools/types.py:1096]: Done, please check again.

**Review by jjmurre — APPROVED** [review]
LGTM

**Pull request merged**